packages/xo-web/src/xo-app/xoa/support/index.js

```diff
@@ -46,6 +47,15 @@ const Support = decorate([
           </Col>
         </Row>
       )}
+      <Row>
+        <Col>
+          <p>
+            <a href='https://support.vates.fr/#/createTicket'>
```

Comment:
Please use [reportOnSupportPanel](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/report-bug-button.js#L55).  This function adds some additional files which give us an information about the health of the user's `XOA`.